backup/bin/barman-parse-metrics

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+"""Usage:
+  barman-parse-metrics <path>
+"""
+
+import docopt
+import json
+import os
+import re
+import tempfile
+
+from datetime import datetime, timezone
+from dateutil import parser
+
+
+# These should come from Alex's module code
+def parse_timestamp(raw):
+    if raw and raw != "None":
+        return parser.parse(raw)
+    else:
+        return None
+
+
+def seconds_elapsed_since(timestamp):
+    if timestamp:
+        now = datetime.now()
+        return (now - timestamp).total_seconds()
+    else:
+        return None
+
+
+def clean_key(k, prefix):
+    return prefix + re.sub("[^A-Za-z0-9]", "_", k)
```

Comment:
Maybe change this to `[^A-Za-z0-9]+`, so that multiple illegal characters collapse to 1 underscore?